WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/ProductPriceSettingsViewModel.swift

```diff
@@ -236,6 +237,11 @@ extension ProductPriceSettingsViewModel: ProductPriceSettingsActionHandler {
     // MARK: - Navigation actions
 
     func completeUpdating(onCompletion: ProductPriceSettingsViewController.Completion, onError: (ProductPriceSetingsError) -> Void) {
+        // Check if sale price was set if there is a sale to be created
+        if dateOnSaleStart != nil && dateOnSaleEnd != nil && getDecimalPrice(salePrice) == nil {
+            onError(.newSaleWithEmptySalePrice)
+            return
+        }
```

Comment:
I think this can be rephrased to something like 
```swift
guard doesScheduleDateHasPrice() {
     return onError(.newSaleWithEmptySalePrice)
}
```

Which is a tad bit easier to read, what do you think?

Reply:
Sure thing! I'll be changing this and I will also remove that comment 😉